genai-perf/README.md

```diff
@@ -73,45 +73,34 @@ INSTALLATION
 
 ## Installation
 
-The easiest way to install GenAI-Perf is through
-[Triton Server SDK container](https://ngc.nvidia.com/catalog/containers/nvidia:tritonserver).
-Install the latest release using the following command:
+The easiest way to install GenAI-Perf is through pip.
+### Install Perf Analyzer (Ubuntu 24.04, Python 3.10+)
 
 ```bash
-export RELEASE="24.10"
-
-docker run -it --net=host --gpus=all  nvcr.io/nvidia/tritonserver:${RELEASE}-py3-sdk
-
-# Check out genai_perf command inside the container:
-genai-perf --help
+pip install git+https://github.com/triton-inference-server/perf_analyzer.git#subdirectory=genai-perf
```

Comment:
I think we should be telling people to use the pypi.org option since that has existed since mid november

```bash
pip install genai-perf
```

https://pypi.org/project/genai-perf/

Reply:
oh definitely. ill update

Reply:
done